### DISCUSSION TITLE
Not start debugging NodeJS

### DISCUSSION THREAD

**Discussion by hopyrez** [apache/netbeans#3632] [Q&A]
I can't start debugging the project NodeJS from Netbeans.
Netbeans write error: /usr/bin/node: [DEP0062]: `node --debug` and `node --debug-brk` are invalid. Please use `node --inspect` and `node --inspect-brk` instead.
Where set up in Netbeans: `node --inspect` and `node --inspect-brk` ?

**Comment by Chris2011** [comment]
Unfortunately, those flags are set hardcoded and was working for nodejs in earlier verison. I think before 8 or smth like that. Atm you can't use debugging NetBeans. Maybe there is a workaround for that but I dunno. https://issues.apache.org/jira/browse/NETBEANS-1320 here you can read more about the problem.
